apps/dashboard/src/components/editor/settings.js

```diff
@@ -29,7 +29,7 @@ export const editorSettings = {
 			fixedToolbar: false,
 		},
 		toolbar: {
-			documentInspector: __( 'Document', 'dashboard' ),
+			documentInspector: __( 'Project', 'dashboard' ),
```

Comment:
Technically, this doesn't seem to be valid configuration anymore and the recent version of Automattic/isolated-block-editor expects a bool.  
Let's leave it for now and raise an issue there.